content/widgets/pagination.md

```diff
@@ -3,32 +3,34 @@ title: Pagination
 type: widget
 html: |
   <div class="ais-Pagination">
-    <ul class="ais-Pagination-list">
-      <li class="ais-Pagination-item ais-Pagination-item--firstPage ais-Pagination-item--disabled">
-        <span class="ais-Pagination-link" aria-label="First">‹‹</span>
-      </li>
-      <li class="ais-Pagination-item ais-Pagination-item--previousPage ais-Pagination-item--disabled">
-        <span class="ais-Pagination-link" aria-label="Previous">‹</span>
-      </li>
-      <li class="ais-Pagination-item ais-Pagination-item--page ais-Pagination-item--selected">
-        <a class="ais-Pagination-link" href="#">1</a>
-      </li>
-      <li class="ais-Pagination-item ais-Pagination-item--page">
-        <a class="ais-Pagination-link" href="#">2</a>
-      </li>
-      <li class="ais-Pagination-item ais-Pagination-item--page">
-        <a class="ais-Pagination-link" href="#">3</a>
-      </li>
-      <li class="ais-Pagination-item">
-        <a class="ais-Pagination-link" href="#">4</a>
-      </li>
-      <li class="ais-Pagination-item ais-Pagination-item--nextPage">
-        <a class="ais-Pagination-link" aria-label="Next" href="#">›</a>
-      </li>
-      <li class="ais-Pagination-item ais-Pagination-item--lastPage">
-        <a class="ais-Pagination-link" aria-label="Last" href="#">››</a>
-      </li>
-    </ul>
+    <nav class="ais-Pagination-nav" aria-label="Pagination">
+      <ul class="ais-Pagination-list">
+        <li class="ais-Pagination-item ais-Pagination-item--firstPage ais-Pagination-item--disabled">
+          <span class="ais-Pagination-link" aria-label="First">‹‹</span>
+        </li>
+        <li class="ais-Pagination-item ais-Pagination-item--previousPage ais-Pagination-item--disabled">
+          <span class="ais-Pagination-link" aria-label="Previous">‹</span>
+        </li>
+        <li class="ais-Pagination-item ais-Pagination-item--page ais-Pagination-item--selected">
+          <span class="ais-Pagination-link" aria-label="Page 1, Current page" aria-current="page">1</span>
```

Comment:
According to the [spec](https://www.a11ymatters.com/pattern/pagination/#indicate-which-element-is-currently-active) you linked this is a booleanish.

```suggestion
          <span class="ais-Pagination-link" aria-label="Page 1, Current page" aria-current="true">1</span>
```

Reply:
According to https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current (and the comments on that article), it should be `aria-current=page` :)

Reply:
Ah, competing standards! Let's go with MDN then 👍